### PR TITLE
tfix

### PR DESCRIPTION
**Pull request by DavidPrevot**

### CODE REVIEW

File: src/Intl/Icu/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@ Symfony Polyfill / Intl: ICU
 ============================
 
 This component maps a collection of functions/classes using the
-[`symfony/intl`](https://github.com/symfony/intl) package in the abscense of the
+[`symfony/intl`](https://github.com/symfony/intl) package in the absence of the
```
Comment (fabpot) [inline src/Intl/Icu/README.md]: I think `absence` is not the right word here. I would rephrase the sentence to something along the lines of:

`This component maps a collection of functions/classes using the intl package when the intl extension is not installed, including:`

ping @weaverryan

Comment (weaverryan) [inline src/Intl/Icu/README.md]: I agree! And unless I'm missing some important meaning, I would also change `maps` => `provides`

**Comment by nicolas-grekas** [comment]
Thank you @DavidPrevot.
